packages/lesswrong/lib/collections/helpers/arbitalLinkedPagesField.ts

```diff
@@ -227,14 +228,35 @@ export function arbitalLinkedPagesField(options: ArbitalLinkedPagesFieldOptions)
         {
           childDocumentId: docId,
           type: 'parent-taught-by-child',
-          isStrong: true,
         },
         { projection: { parentDocumentId: 1 } }
       ).fetch();
 
 
       const teachesDocumentIds = teachesRels.map((rel) => rel.parentDocumentId);
 
+      // Step X: Fetch parent relationships
+      const parentRels = await ArbitalTagContentRels.find(
+        {
+          childDocumentId: docId,
+          type: 'parent-is-parent-of-child',
+        },
+        { projection: { parentDocumentId: 1 } }
+      ).fetch();
+
+      const parentDocumentIds = parentRels.map((rel) => rel.parentDocumentId);
+
+      // Step Y: Fetch child relationships
+      const childRels = await ArbitalTagContentRels.find(
+        {
+          parentDocumentId: docId,
+          type: 'parent-is-parent-of-child',
+        },
+        { projection: { childDocumentId: 1 } }
+      ).fetch();
```

Comment:
x & y?

Also surely these could be parallelized.

Reply:
actually why aren't requirementsRels, teachesRels, and parentRels just a single query with `type: { $in: ['parent-is-requirement-of-child', 'parent-taught-by-child', 'parent-is-parent-of-child'] }`?  (In parallel with fetching the childRels.)

Reply:
Addressed